fapolicy_analyzer/ui/policy_rules_admin_page.py

```diff
@@ -105,7 +105,6 @@ def __init__(self, audit_file: str = None):
             details_widget_name="objectDetails",
         )
         object_tabs.append_page(self.object_list.get_ref(), Gtk.Label(label="Object"))
-
```

Comment:
Looks like the only changes to this file are removing a couple returns.  Please revert this if there aren't any code changes.